x-pack/plugins/infra/server/graphql/log_entries/schema.gql.ts

```diff
@@ -90,6 +90,10 @@ export const logEntriesSchema = gql`
     id: ID!
     "The index where the document was found"
     index: String!
+    "The timestamp value for the document - derived from the source configuration timestamp setting"
+    timestamp: String
+    "The tiebreaker value for the document - derived from the source configuration tiebreaker setting"
+    tiebreaker: Int
```

Comment:
How about we re-use the common `InfraTimeKey` type for a `key` field like in most other log-related apis? That key could be non-optional and use the usual normalized values (`timestamp` as a `Float`, see the comments further below).